.github/bors_log_expected_warnings

```diff
@@ -148,3 +148,125 @@ gengtype-lex.cc:357:15: warning: this statement may fall through [-Wimplicit-fal
+--../../libcpp/macro.cc:3719:58: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/analyzer/program-state.cc:1330:15: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/c-family/c-common.cc:6595:14: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/c-family/c-common.cc:6599:16: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/c/c-convert.cc:84:13: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/c/c-typeck.cc:11946:16: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/c/c-typeck.cc:3730:17: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/c/c-typeck.cc:4500:16: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/c/c-typeck.cc:6687:23: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/collect-utils.cc:237:21: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/collect2.cc:2397:21: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/diagnostic.cc:2206:15: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/fold-const.cc:313:13: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/gimple-range-gori.cc:318:19: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/ipa-devirt.cc:965:10: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/lto-wrapper.cc:1365:10: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/lto-wrapper.cc:1957:23: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../gcc/tree-ssa-sccvn.cc:280:21: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../libcpp/expr.cc:821:35: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../libcpp/expr.cc:824:38: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../libcpp/expr.cc:834:33: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../libcpp/macro.cc:185:26: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../libcpp/macro.cc:214:34: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../libcpp/macro.cc:3704:25: warning: format not a string literal and no format arguments [-Wformat-security]
+-+../../libcpp/macro.cc:3719:25: warning: format not a string literal and no format arguments [-Wformat-security]
```

Comment:
You need a newline at the end of this file

Reply:
thanks I was stuck on this one.